src/libsyntax_pos/lib.rs

```diff
@@ -491,6 +493,8 @@ impl Decodable for FileMap {
             Ok(FileMap {
                 name: name,
                 name_was_remapped: name_was_remapped,
+                // `crate_of_origin` has to be set by the importer.
+                crate_of_origin: 0xEFFF_FFFF,
```

Comment:
Sorry :)
It's a number that is unlikely to ever be a valid `CrateNum`. `0xFFFF_FFFF` is already taken for some other special purpose. I should introduce an `INVALID_CRATE` const.